neo4j/bookmarks_test.go

```diff
@@ -255,10 +258,10 @@ type simpleBookmarkSupplier struct {
 	databaseBookmarks func(string) neo4j.Bookmarks
 }
 
-func (s *simpleBookmarkSupplier) GetAllBookmarks() neo4j.Bookmarks {
+func (s *simpleBookmarkSupplier) GetAllBookmarks(context.Context) neo4j.Bookmarks {
 	return s.allBookmarks()
 }
 
-func (s *simpleBookmarkSupplier) GetBookmarks(db string) neo4j.Bookmarks {
+func (s *simpleBookmarkSupplier) GetBookmarks(_ context.Context, db string) neo4j.Bookmarks {
 	return s.databaseBookmarks(db)
 }
```

Comment:
Not sure if I missed it: are there tests that assert the the BMM actually passes the right context down to the supplier and notifier?

Reply:
nope, good idea

Reply:
fixed in 6dbb4a44db87f6da8a456d01d706cae9c8c8a025